### PR TITLE
Added notice to contribution guidelines

### PR DESCRIPTION
**Pull request by JosXa**
We are getting a lot of requests for different libraries and PLs. This is an effort to change that

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+IMPORTANT NOTICE FOR NEW CONTRIBUTORS: This is the repository of the Python library 
+"python-telegram-bot". We do not respond to questions in other programming languages 
+or libraries such as "telepot" or "pyTelegramBotAPI".
+
 Thanks for reporting issues of python-telegram-bot!
 
 Use this template to notify us if you found a bug, or if you want to request a new feature.
 If you're looking for help with programming your bot using our library, feel free to ask your
 questions in out telegram group at: https://t.me/pythontelegrambotgroup
+We also get a lot of issues that could be answered by a quick question in the group, so please join us on Telegram.
```
Comment (Eldinnie) [inline .github/ISSUE_TEMPLATE.md]: This is practically the same as the line above,

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 <!--
+IMPORTANT NOTICE FOR NEW CONTRIBUTORS: This is the repository of the Python library
```
Comment (Eldinnie) [inline .github/ISSUE_TEMPLATE.md]: This is the issue template. Not for contributers right?

Comment (JosXa) [reply .github/ISSUE_TEMPLATE.md]: Wasn't sure what to call it :/ You got an idea?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (JosXa) [reply .github/ISSUE_TEMPLATE.md]: Sure, but people obviously don't follow the advise. So I wanted to emphasize it even more

**Comment by Eldinnie** [comment]
As discussed, people don't read, no matter how much text you put in the template. Closing

**Pull request closed without merging**